background/app.js

```diff
@@ -0,0 +1,25 @@
+import "../modules/tab.prototype.js"
+import * as storage from "../modules/storage.js"
+import * as tabs from "../modules/tabs.js"
+import * as page from "../modules/page.js"
+import * as extension from "../modules/extension.js"
+
+extension.onInstalled(() => {
+  extension.createContextMenus({
+    title: 'Read later',
```

Comment:
All files should consist with single quote or double quote. Recommend to use eslint.